### PR TITLE
adding 121 glossary

### PR DESCRIPTION
**Pull request by cassandradamico**

### CODE REVIEW

File: docs/glossary-121.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+- [Payment Overview](#payment-overview)
+- [Payment Transfer statuses](#payment-transfer-statuses)
+
+### General terms
+
+| Term                   | Definition                           |
+| :------------------    | :------------------------------------------------------------------- |
+| **121 Platform**       | (“one to one”) is a platform that helps project managers by increasing efficiency in setting up and executing a cash program Information Management (IM) system.  |
+| **CVA**                | Cash and Voucher Assistance, referring to Cash Transfer Programming (CTP). |
+| **Project**            | A program created by users in the platform. |
```
Comment (tijsziere) [inline docs/glossary-121.md:22]: definition should not say 'program', we're defining a project here

File: docs/glossary-121.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+| **FSP**                | Financial Service Provider - an entity that provides financial services, which may include digital payment services.|
+| **Budget**             | Amount of money in the project that will be disbursed to included people/households.|
+| **Registrations**      | People or households that registered, subscribed to a project or programme.|
```
Comment (tijsziere) [inline docs/glossary-121.md:25]: I dont think we should mention programme here, and if we are: be consistent in program vs. programme

File: docs/glossary-121.md [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+| **Type / Select value**| when a column is selected to filter by, users can then type or select the word/ numerical value they wish to filter the table by.|
+| **Log in**             | How a user can access their account.|
+
+### Tab Titles
+
+| Term                   | Definition                           |
+| :------------------    | :------------------------------------------------------------------- |
+| **Home** | The main page to overview all active programs. The Admin can oversee all programs. A user assigned to programs can only see the specific programs.|
+| **Active** | A program that is currently running.|
+| **Archive** | A program that has ended but the information is archived by an admin. (It cannot be run again but the user can read them and copy them into new program).|
```
Comment (tijsziere) [inline docs/glossary-121.md:41]: This is currently not functionality we support

File: docs/glossary-121.md [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+| **Overview**| The information and metrics about a program.|
+| **Team** | 121 users that will be working on this program and their assigned roles. |
+| **People Affected** | All registrations are listed in this page. The registration and inclusion or rejection of beneficiaries in the program will be displayed in this page under different status: registered, validated, included, declined, completed.|
+| **Payment** | The page were users can have an overview of the payment status and/or execute payment tranches towards included beneficiaries.|
+
+### Program Cards
+
+| Term                   | Definition                           |
+| :------------------    | :------------------------------------------------------------------- |
+| **Program Cards** | A small window providing an brief overview of the program details. |
```
Comment (tijsziere) [inline docs/glossary-121.md:54]: This is not terminology relevant to the user, i think

File: docs/glossary-121.md [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+| **Program title** | The name of the program. |
+| **Last updated** | The most recent date and time when the system checked for changes made to a program.|
+| **Active** | Programs that are being created or already running.|
+| **Archived** | Rather than deleting, the user can archive a project, it is not active but its details are still available to view and can be copied.|
+
+### Registration Status
+
+| Term                   | Definition                           |
+| :------------------    | :------------------------------------------------------------------- |
+| **People Affected, PA** | Registered beneficiaries or households in the program, also referred to PA in 121. Depending on the SOPs, the status of the PA can vary: new, registered, validated, included, declined, completed, paused.|
```
Comment (tijsziere) [inline docs/glossary-121.md:68]: I believed we were moving away from People Affected. It is currently in the product though, not sure on how to proceed. @cassandradamico what do you think?

Comment (tijsziere) [reply docs/glossary-121.md:68]: People Affected, PA is also not a Registration status, so it is in the wrong section anyway

**Review by tijsziere — CHANGES_REQUESTED** [review]

File: docs/glossary-121.md (hunk 5 above)
Comment (cassandradamico) [reply docs/glossary-121.md:68]: @tijsziere I will keep PA for now, as this is not used outside of RCRC movement, for understanding matter. Will change whenever the terminilogy change in 121

**Comment by cassandradamico** [comment]
Changes accepted + merged
but it does not work. 
I can only ask you to check the changes again and cannot make an approval.
Sorry

**Review by tijsziere — APPROVED** [review]

**Pull request merged**